prow/clonerefs/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// clonerefs is a library for cloning references
```

Comment:
Is this file really necessary?

Reply:
Someone is going to `/lint` this PR ... I tried to pre-empt that :)

Reply:
someone indeed 😈

Reply:
we still have a decent number of golint issues in the repo 🤷‍♂️